iron-image.html

```diff
@@ -355,21 +354,16 @@
         this.style.height = isNaN(this.height) ? this.height : this.height + 'px';
       },
 
-      _preventLoadChanged: function() {
-        if (this.preventLoad || this.loaded) return;
-
-        this._reset();
```

Comment:
This check for `this.loaded` and calling `this._reset()` seems to be gone. I don't know anything about this code -- is it still needed?